debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/IdStrategy.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.bson.BsonValue;
+
+import io.debezium.connector.mongodb.converters.sink.SinkDocument;
+
+public interface IdStrategy {
```

Comment:
Please add JavaDoc. Is this something tht is intended to be configurable in future using custom class?

Reply:
done